### DISCUSSION TITLE
Jira and Bitbucket VS Code Extension (unable to verify the first certificate)

### DISCUSSION THREAD

**Discussion by jamekaze** [microsoft/vscode-discussions#1354] [Extension Development QnA] [answered]
Hello,

 

A user at my company is trying to make use of the Jira and Bitbucket extension for VS Code made by Atlassian Labs (linked below). When trying to make the connection to either of our Jira or Bitbucket instances (Data Center) we both get the below error message.

Seems to me that either VS Code or the Jira/Bitbucket extension is not using the correct certificate store to validate the web cert that our applications are using. Does any one know how to properly configure VS Code to resolve this issue?

Thanks in advance!
![vscode_firstcertificate](https://github.com/user-attachments/assets/269decdd-562b-4806-aa8c-de9a6f170520)

**Comment by AhmedMagdy2511** [comment]
Hello jamekaze,

you can check this article and try it:
https://confluence.atlassian.com/jirakb/unable-to-authenticate-to-jira-server-dc-from-the-atlassian-vs-code-extension-1051983995.html

**Comment by gjsjohnmurray** [comment, marked as answer]
Please also note for future reference that this forum category is called "Extension **Development** QnA". This whole forum currently focuses on peer support among **developers** of VS Code extensions rather than on assisting users of extensions.

More info is in this "READ FIRST" post pinned at the top of the forum.

https://github.com/microsoft/vscode-discussions/discussions/5
